File: android/src/main/kotlin/com/syncrhrohealth/bandpass_ble_connection/core/command/HotBoxDataGetCmd.kt
package com.syncrhrohealth.bandpass_ble_connection.core.command

import com.syncrhrohealth.bandpass_ble_connection.core.command.builder.CommandBuilder
import com.syncrhrohealth.bandpass_ble_connection.core.enum.CommandCode
import com.syncrhrohealth.bandpass_ble_connection.core.handler.DeviceHandler


object HotBoxDataGetCmd {
    fun send(handler: DeviceHandler) {
        val builder = CommandBuilder.build(CommandCode.PACKET_CMD_HOTBOX_DATA_GET)
        handler.write(builder)
    }
}

File: ios/Classes/Core/Command/HotBoxDataGetCmd.swift
//
//  HotBoxDataGetCmd.swift
//  Runner
//
//  Created by MAC on 11/3/25.
//

import Foundation

struct HotBoxDataGetCmd {
    static func send(handler: DeviceHandler) {
        let builder = CommandBuilder.build(commandCode: .packetCmdHotboxDataGet)
        handler.write(data: builder)
    }
}
